### ISSUE TITLE
Bug: `operator-no-unspaced`: false positive for CSS `type()`

### ISSUE THREAD

**Issue by Arkellys** [stylelint-scss/stylelint-scss#1310]
The rule `operator-no-unspaced` returns false positive when using CSS [`type()`](https://developer.mozilla.org/en-US/docs/Web/CSS/Reference/Values/type):

```css
attr(data-color type(<color>))
/*                   ↑     ↑
Expected single space before ">"
Expected single space after ">"
*/
```

**Comment by kristerkari** [comment]
Thanks for reporting!
